index.html

```diff
@@ -45,71 +90,73 @@ <h2 class="header__time">2022.05.16(MON) ~ 20(FRI)</h2>
             <h2>Main Program</h2>
             <hr class="line">
         </div>
+        <div class="desktop-grid">
+            <div class="Main-program__cards">
 
-        <div class="Main-program__cards">
+                <div class="card__left">
+                    <img src="./img/lecture.png" alt="Scratch">
+                    <p>Lecturer</p>
+                </div>
+                <div class="card__right">
+                    <p>Listen to the speakers from the various countries about the message of sharing and opening
+                    </p>
+                </div>
 
-            <div class="card__left">
-                <img src="./img/lecture.png" alt="Scratch">
-                <p>Lecturer</p>
-            </div>
-            <div class="card__right">
-                <p>Listen to the speakers from the various countries about the message of sharing and opening
-                </p>
             </div>
 
-        </div>
 
+            <div class="Main-program__cards">
```

Comment:
Please be sure that all of your images/icons are being loaded properly 🧐 

![image](https://user-images.githubusercontent.com/15807343/164560246-31d85f91-3c3d-4b05-a78b-cea8ed822117.png)

The same issue is happening with your speakers 👀

![image](https://user-images.githubusercontent.com/15807343/164560295-ec9dfdb3-7442-4365-b7aa-3ba6adb98333.png)